### ISSUE TITLE
ZAP API scan - Ability to mention scheme from command prompt to override from swagger file

### ISSUE THREAD

**Issue by SudheendraSingh** [zaproxy/zaproxy#5510]
**Is your feature request related to a problem? Please describe.**
My swagger file has
schemes:
  - https
  - http

I want to only use http for my ZAP API docker scanning, but it takes https scheme from the swagger file.

**Describe the solution you'd like**
It would be great to have a command line option similar to override the scheme of the actual API URL to scan, which is similar to -O (used to override host in swagger)

**Describe alternatives you've considered**
Alternately, I would need to edit the swagger just to run the ZAP API scan.

**Additional context**

**Comment by thc202** [comment]
This will require changes in the add-on first.

**Comment by vinaymbhoj** [comment]
Yes , we would really like to have this feature of override https schema as it is very much required for few of the micro services in our project .

**Comment by SudheendraSingh** [comment]
Can this feature be implemented soon or any indication of timeline? This would help me to push the ZAP API scanner across the organisation.

**Comment by kingthorin** [comment]
As far as I know it isn’t on the radar for any of the core team, however, anyone is welcome to tackle the change. We’re here to advise if needed.

**Comment by SudheendraSingh** [comment]
Thanks for letting me know. I will continue with a workaround.
